global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.0.100-preview3-010431"
```

Comment:
Update to latest official preview: 3.0.100-preview5-011568

Reply:
Done